SilMock/Google/Service/Directory.php

```diff
@@ -39,5 +43,17 @@ public function __construct($client, ?string $dbFile = null)
         $this->users_aliases = new UsersAliasesResource($dbFile);
         $this->verificationCodes = new VerificationCodesResource($dbFile);
         $this->twoStepVerification = new TwoStepVerification($dbFile);
+        $this->client = new Client();
+        Assert::notEmpty($client, 'Expecting a client to be passed!');
+    }
+
+    public function getClient()
```

Comment:
```suggestion
    public function getClient(): Client
```

Reply:
no, because I am trying to match the google php sdk functions.